autotest/utilities/test_gdalalg_raster_create.py

```diff
@@ -524,3 +524,26 @@ def test_gdalalg_raster_create_empty_bbox():
         match="0 value has been specified for argument 'bbox', whereas exactly 4 were expected",
     ):
         alg.Run()
+
+
+def test_gdalalg_raster_create_pipeline_first_step():
+
+    with gdal.Run(
+        "raster pipeline",
+        output_format="MEM",
+        pipeline="create --bbox -10,-10,10,10 --size 2,2 ! write",
+    ) as alg:
+        assert alg.Output().GetGeoTransform() == (-10, 10, 0, 10, 0, -10)
+
+
+def test_gdalalg_raster_create_pipeline_middle_step():
+
+    with gdal.Run(
+        "raster pipeline",
+        output_format="MEM",
+        pipeline="read ../gcore/data/byte.tif ! create --burn 7 ! write",
```

Comment:
I don't totally understand this pipeline. Is the "byte.tif" used as  `--like` for create somehow automatically? Is that behavior documented?